includes/class-event.php

```diff
@@ -22,6 +22,9 @@ class Event {
 	// When the event will run next.
 	private int $timestamp;
 
+	private $created;
+	private $last_modified;
```

Comment:
Need to set these when we pull from the database for `get_legacy_event_format()` (legacy compatibility w/ hook params).

Will probably make them first-class citizens if we end up using them elsewhere.